library/x509_crt.c

```diff
@@ -1449,8 +1449,8 @@ int mbedtls_x509_crt_parse_path(mbedtls_x509_crt *chain, const char *path)
         }
 
         w_ret = WideCharToMultiByte(CP_ACP, 0, file_data.cFileName,
-                                    lstrlenW(file_data.cFileName),
-                                    p, (int) len - 1,
```

Comment:
Previously, we passed in the number of characters excluding the terminating null character, and had filled the buffer with `0` bytes to ensure the output is null-terminated.